api/docker/confd/templates/parameters.yml.tmpl

```diff
@@ -3,6 +3,18 @@
 
 # This file is a "template" of what your parameters.yml file should look like
 parameters:
+    cloudwatch_logs_client_params:
+       version: 'latest'
+       region: 'eu-west-1'
+{{ if exists "/cloudwatch/logs/localstack" }}
```

Comment:
Could you explain this part for me please? When would this exist and when wouldn't it?

Reply:
So we use confd to handle some of the config values here in different environments. I'm not a huge fan of it as the syntax isn't very clear with what its doing and Symfony has its own environment system that we can hook into with things like `env.test` and `config.test` etc but that's a different discussion 😂 

This is basically saying if there's an environment variable called CLOUDWATCH_LOGS_LOCALSTACK then compile the values down. We only have this variable in environments where we want localstack to run so its excluded any places that use real infra. The confd process itself runs after [composer is run](https://github.com/ministryofjustice/opg-digideps/blob/main/client/composer.json#L38) as a post hook and is something we'll need to update when we do the Symfony upgrade as the package that handles that won't work in Symfony 4+ seeing as they use [Symfony Flex](https://symfony.com/doc/current/setup/flex.html) instead.

Reply:
Ah! okay that makes more sense to me now 👍🏻